lib/engine/step/g_1860/dividend.rb

```diff
@@ -27,7 +27,7 @@ def process_dividend(action)
 
           entity.operating_history[[@game.turn, @round.round_num]] = OperatingInfo.new(
             routes,
-            action,
+            (@game.insolvent?(entity) ? @game.actions.last : action),
```

Comment:
Why are you doing this? An insolvent company has a withhold dividend action. Why is this needed?

Reply:
It has a phantom action that doesn't have an id

Reply:
Maybe a dumb question, but why can't 1860 use the same logic as `Dividend::skip!`? Why is the phantom action needed here but not there?

Reply:
Because unlike normal dividend skip operations, there is a route being run for an insolvent company, but the corp is forced to withhold (and the amount isn't zero).

Reply:
Who is using the id on the actions stored in operating_history?

Reply:
meant to put this here, but you can jump to an action in the spreadsheet which is why it needs an id

Reply:
I think I'm OK with these changes then.

Reply:
What revenue gets reported on the spreadsheet?